### PR TITLE
add inventory plugin unit test `test_verify_file`

### PR DESCRIPTION
**Pull request by keilr**
##### SUMMARY
This PR adds a positive unit test for the inventory plugin function `verify_file`. So far there was only a negative test (non-existing config file)

##### ISSUE TYPE
- Test Pull Request

##### COMPONENT NAME
- tests/unit/plugins/inventory/test_cobbler.py
- tests/unit/plugins/inventory/test_linode.py
- tests/unit/plugins/inventory/test_lxd.py
- tests/unit/plugins/inventory/test_proxmox.py
- tests/unit/plugins/inventory/test_stackpath_compute.py

### CODE REVIEW

File: tests/unit/plugins/inventory/test_lxd.py [hunk 1]
```diff
@@ -51,8 +51,14 @@ def inventory():
...
 
+def test_verify_file(tmp_path, inventory):
+    file = tmp_path / "foobar.lxd.yml"
+    file.touch()
+    assert inventory.verify_file(str(file)) is True
+
+
 def test_verify_file_bad_config(inventory):
-    assert inventory.verify_file('foobar.lxd.yml') is False
+    assert inventory.verify_file('foobar.lsd.yml') is False
```
Comment (felixfontein) [inline tests/unit/plugins/inventory/test_lxd.py:61]: Why this change?

File: tests/unit/plugins/inventory/test_proxmox.py [hunk 2]
```diff
@@ -21,8 +21,14 @@ def inventory():
...
 
+def test_verify_file(tmp_path, inventory):
+    file = tmp_path / "foobar.proxmox.yml"
+    file.touch()
+    assert inventory.verify_file(str(file)) is True
+
+
 def test_verify_file_bad_config(inventory):
-    assert inventory.verify_file('foobar.proxmox.yml') is False
+    assert inventory.verify_file('foobar.pve.yml') is False
```
Comment (felixfontein) [inline tests/unit/plugins/inventory/test_proxmox.py:31]: Same question here.

File: tests/unit/plugins/inventory/test_stackpath_compute.py [hunk 3]
```diff
@@ -66,8 +66,14 @@ def test_get_stack_slugs(inventory):
...
 
+def test_verify_file(tmp_path, inventory):
+    file = tmp_path / "foobar.stackpath_compute.yml"
+    file.touch()
+    assert inventory.verify_file(str(file)) is True
+
+
 def test_verify_file_bad_config(inventory):
-    assert inventory.verify_file('foobar.stackpath_compute.yml') is False
+    assert inventory.verify_file('foobar.stockpath_compute.yml') is False
```
Comment (felixfontein) [inline tests/unit/plugins/inventory/test_stackpath_compute.py:76]: Same question here.

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! I have a few questions on some changes to existing tests, though:

File: tests/unit/plugins/inventory/test_lxd.py (hunk 1 above)
Comment (keilr) [reply tests/unit/plugins/inventory/test_lxd.py]: Thanks for the review. I tried to align the `test_verify_file_bad_config` test on all inventory plugins. at the moment some use typos, others don't. I'll remove this changes from this PR.

File: tests/unit/plugins/inventory/test_cobbler.py [hunk 4]
```diff
@@ -37,5 +37,11 @@ def test_init_cache(inventory):
...
 
 
+def test_verify_file(tmp_path, inventory):
+    file = tmp_path / "foobar.cobbler.yml"
+    file.touch()
+    assert inventory.verify_file(str(file)) is True
+
+
 def test_verify_file_bad_config(inventory):
     assert inventory.verify_file('foobar.cobber.yml') is False
```
Comment (felixfontein) [inline tests/unit/plugins/inventory/test_cobbler.py:47]:
```suggestion
    assert inventory.verify_file('foobar.cobbler.yml') is False
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@keilr thanks a lot for improving the tests!
